### PR TITLE
Update readme and samples for iOS

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: Gradle/HelloFX/README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # Gluon Client HelloFX using Gradle
 
-This is the most basic sample for running JavaFX samples on a client that run with OpenJDK 11, GraalVM and the Gluon 
-Client plugin for [Gradle](https://github.com/gluonhq/client-gradle-plugin/).
-We recommend you first deploy the HelloWorld sample located in the same repository as this HelloFX sample.
+This is the most basic sample for running JavaFX on a client with OpenJDK 11, GraalVM and the Gluon 
+[Client plugin for Gradle](https://github.com/gluonhq/client-gradle-plugin/).
```
Comment (johanvos) [inline Gradle/HelloFX/README.md]: put brackets before [Gluon Client]

File: Gradle/HelloWorld/README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Gluon Client HelloWorld using Gradle
 
 This is the most basic sample for running Java and JavaFX samples on a client that run with OpenJDK 11, GraalVM and the Gluon
```
Comment (abhinayagarwal) [inline Gradle/HelloWorld/README.md]: This sample is linked fromHelloFX ReadME as well. The readme for this sample should not mention JavaFX IMO. 

> ...for running Java on client with...

File: Gradle/HelloFX/README.md [hunk 3]
```diff
@@ -17,6 +19,7 @@ To compile and link:
     ./gradlew clean nativeBuild
     
 This task performs 3 steps: 
+
 * it will compile the Java source code into Java bytecode (done by the subtask `build`)
 * it will compile the Java bytecode to native code (done by the subtask `nativeCompile`)
 * it will link the native code with dependencies and required VM code into an executable (done by the subtask `nativeLink`)
```
Comment (abhinayagarwal) [inline Gradle/HelloFX/README.md]: "it will" is redundant here :)

File: Gradle/HelloWorld/README.md [hunk 4]
```diff
@@ -16,6 +16,7 @@ To compile and link:
     ./gradlew clean nativeBuild
     
 This task performs 3 steps: 
+
 * it will compile the Java source code into Java bytecode (done by the subtask `build`)
 * it will compile the Java bytecode to native code (done by the subtask `nativeCompile`)
 * it will link the native code with dependencies and required VM code into an executable (done by the subtask `nativeLink`)
```
Comment (abhinayagarwal) [inline Gradle/HelloWorld/README.md]: Same here

File: Gradle/HelloWorld/README.md [hunk 5]
```diff
@@ -33,28 +34,30 @@ To run the generated executable:
 Before you can use the plugin for deploying to iOS devices, you need to have `llvm` in your path. You can download llvm for 
 mac <a href="http://releases.llvm.org/6.0.0/clang+llvm-6.0.0-x86_64-apple-darwin.tar.xz">here</a>.
 
-The only required change in the 'build.gradle' is providing the optional `target` property in the `gluonClient` configuration inside the `build.gradle` file:
+The only required change in the `build.gradle is providing the optional `target` property in the `gluonClient` configuration inside the `build.gradle` file:
```
Comment (abhinayagarwal) [inline Gradle/HelloWorld/README.md]: `build.gradle` isn't closed with "`"

File: Maven/HelloFX/README.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+To compile and link:
+
+    mvn clean client:build
+    
+This task performs 3 steps: 
+
+* it will compile the Java source code into Java bytecode (done by the subtask `client:compile`)
+* it will compile the Java bytecode to native code (done by the subtask `client:compile`)
+* it will link the native code with dependencies and required VM code into an executable (done by the subtask `client:link`)
```
Comment (abhinayagarwal) [inline Maven/HelloFX/README.md]: same here

File: Maven/HelloFXML/README.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+To compile and link:
+
+    mvn clean client:build
+    
+This task performs 3 steps: 
+
+* it will compile the Java source code into Java bytecode (done by the subtask `client:compile`)
+* it will compile the Java bytecode to native code (done by the subtask `client:compile`)
+* it will link the native code with dependencies and required VM code into an executable (done by the subtask `client:link`)
```
Comment (abhinayagarwal) [inline Maven/HelloFXML/README.md]: same here

File: Maven/HelloFXML/README.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+```
+    public static void main(String[] args) {
+        System.setProperty("os.target", "ios");
+        System.setProperty("os.name", "iOS");
+        System.setProperty("glass.platform", "ios");
+        System.setProperty("targetos.name", "iOS");
+        launch(args);
+    }
+
+```
```
Comment (abhinayagarwal) [inline Maven/HelloFXML/README.md]: same here

File: Maven/HelloFXML/README.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
```
Comment (abhinayagarwal) [inline Maven/HelloFXML/README.md]: And here

File: Maven/HelloFX/README.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+    
+```
+    public static void main(String[] args) {
+        System.setProperty("os.target", "ios");
+        System.setProperty("os.name", "iOS");
+        System.setProperty("glass.platform", "ios");
+        System.setProperty("targetos.name", "iOS");
+        launch(args);
+    }
+
```
Comment (abhinayagarwal) [inline Maven/HelloFX/README.md]: same here

File: Gradle/HelloFX/README.md [hunk 11]
```diff
@@ -56,10 +59,32 @@ future, but for now, modify the `main` method in `HelloFX` as follows
...
+```
+    public static void main(String[] args) {
+        System.setProperty("os.target", "ios");
+        System.setProperty("os.name", "iOS");
+        System.setProperty("glass.platform", "ios");
+        System.setProperty("targetos.name", "iOS");
+        launch(args);
+    }
 
+```
```
Comment (abhinayagarwal) [inline Gradle/HelloFX/README.md]: Remove space or remove ```

File: Maven/HelloFX/README.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+```
```
Comment (abhinayagarwal) [inline Maven/HelloFX/README.md]: same here, remove space or remove ```

**Pull request merged**